projects/control-service/projects/pipelines_control_service/src/integration-test/java/com/vmware/taurus/service/deploy/DataJobDeploymentCrudITV2.java

```diff
@@ -74,6 +73,14 @@ public class DataJobDeploymentCrudITV2 extends BaseIT {
 
   @Autowired private DeploymentService deploymentService;
 
+  private boolean jobEnabled;
```

Comment:
I think I would prefer it if you weren't creating fields here and instead just using local variables.